api/models/process.go

```diff
@@ -294,9 +296,9 @@ func ListPendingProcesses(app string) ([]*Process, error) {
 			tres, err := ECS().DescribeTaskDefinition(&ecs.DescribeTaskDefinitionInput{
 				TaskDefinition: d.TaskDefinition,
 			})
-
 			if err != nil {
-				return nil, err
+				fmt.Printf("ns=kernel at=ListPendingProcesses state=error message=\"%s\"\n", err)
```

Comment:
My line of thinking here was if `GetAppServices()` did fail there little else to do. If one ECS call failed, other could still pass. After looking at it again, I can see how line 282 could be changed to not bail out entirely so that running and one-off processes could still be displayed.